src/modules/translation/languages/en.json

```diff
@@ -134,13 +135,16 @@
     "subtitle_one": "You can create custom layouts for your scene.",
     "subtitle_two": "Enter values in the boxes below for the desired size."
   },
+  "layout_picker": {
+    "rows": "rows",
+    "columns": "columns"
+  },
   "custom_layout": {
     "title": "Custom",
-    "max_area_error": "Your scene can't have more than {area} parcels",
     "up_to": "Up to {max} parcels"
   },
-  "layout_picker": {
-    "rows": "rows",
-    "columns": "columns"
+  "project_layout_picker": {
+    "max_area_error": "Your scene can't have more than {area} parcels",
+    "min_area_error": "Your scene it's too small"
```

Comment:
is!

Reply:
I'd rather get rid of the min error message all together:

![min-erroir](https://user-images.githubusercontent.com/2781777/53571318-608bc400-3b47-11e9-91e3-6817459070ab.gif)

it feels better if the modal doesn't resize, and I think is clear enough that you can't submit with an empty (or zero or negative) field

Reply:
oh! ok
I'll do that